boards/nucleo-l476/include/periph_conf.h

```diff
@@ -234,6 +234,17 @@ static const spi_conf_t spi_config[] = {
 #define DAC_NUMOF           (0)
 /** @} */
 
+/**
+ * @brief   RTT configuration
+ *
+ * On the STM32Lx platforms, we always utilize the LPTIM1.
```

Comment:
Maybe rephrase then: `On the STM32Lx platforms, LPTIM1 is always used` ?

This makes me think that we could probably makes this configurable in the board periph_conf file ? as the stm32l476rg has 2 LPTIM.